README.md

```diff
@@ -100,30 +162,41 @@ This will take approximately a minute to run, with the majority of the time spen
+                                               8545/tcp -> 127.0.0.1:55903
+                                               8546/tcp -> 127.0.0.1:55901
+ethereum-node-1-1649105164   ethereum-node-1   8545/tcp -> 127.0.0.1:55907
+                                               8546/tcp -> 127.0.0.1:55908
+                                               30303/tcp -> 127.0.0.1:55909
+                                               30303/udp -> 127.0.0.1:56275
+ethereum-node-2-1649105167   ethereum-node-2   30303/udp -> 127.0.0.1:51099
+                                               8545/tcp -> 127.0.0.1:55964
+                                               8546/tcp -> 127.0.0.1:55965
+                                               30303/tcp -> 127.0.0.1:55963
 
-```javascript
-Set(3) { 'bootnode', 'ethereum-node-1', 'ethereum-node-2' }
 ```
 
 But what just happened?
 
-Starting networks is a very common task in Kurtosis, so we provide [a framework called "modules"](https://docs.kurtosistech.com/modules.html) for making it dead simple. An executable module is basically a chunk of code that responds to an "execute" command, packaged as a Docker image, that runs inside a Kurtosis enclave - sort of like Docker Compose on steroids. In the steps above, we called `enclaveCtx.loadModule` to load [the Ethereum module](https://github.com/kurtosis-tech/ethereum-kurtosis-module) into the enclave with module ID `eth-module`, and `ethModuleCtx.execute` to run it. The Ethereum module doesn't take any parameters at load or execute time (hence the `{}`), but other modules do.
+Starting networks is a very common task in Kurtosis, so we provide [a framework called "modules"](https://docs.kurtosistech.com/modules.html) for making it dead simple. An executable module is basically a chunk of code that responds to an "execute" command, packaged as a Docker image, that runs inside a Kurtosis enclave - sort of like Docker Compose on steroids. In the steps above, we executed `kurtosis module exec` to load [the Ethereum module](https://github.com/kurtosis-tech/ethereum-kurtosis-module) into a new enclave. The Ethereum module doesn't take any parameters at load or execute time, but other modules do so, you can add them using the `--execute-params` flag (e.g: --execute-params '{"numDatastores":2}')
 
 Now that you have a pet Ethereum network, let's do something with it.
 
 Step Four: Talk To Ethereum (5 minutes)
 ---------------------------------------
-Talking to Ethereum in Javascript is easily accomplished with [the EthersJS library](https://docs.ethers.io/v5/). Your Javascript REPL is running in a Docker image (so that you don't need Javascript installed locally), so we'll need to install EthersJS on that image.
+Talking to Ethereum is easily accomplished with cURL commands.
 
-First, in a new terminal window, run the following to find the enclave our REPL is running inside:
+First, run the following to find the enclave our Eth network is running inside:
```

Comment:
You can remove this with the comment above, where we put the Ethereum module inside the `demo` enclave